.rubocop.yml

```diff
@@ -2,6 +2,7 @@ inherit_from: .rubocop_todo.yml
 
 AllCops:
   NewCops: enable
+  TargetRubyVersion: 2.7
```

Comment:
As Dalton mentioned, we aim for 2.5. Please set it up here as well.

Reply:
We check only 2.6, 2.7 & 3.0. Let's aim for 2.6 and fix the README!